builder-support/specs/dnsdist.spec

```diff
@@ -153,6 +153,10 @@ fi
 %endif
 %if 0%{?rhel} >= 7
 %systemd_post %{name}.service
+if [ $1 -eq 2 ] ; then
```

Comment:
I think this should come before `%systemd_post %{name}.service`

Reply:
note that order does not really matters as long as we are using a check for update as `%systemd_post` translate to `if [ $1 -eq 1 ] ; then ... fi`
I was assuming `preset` was implying a daemon-reload but after reading some docs https://freedesktop.org/wiki/Software/systemd/Preset/ it s mainly going to trigger enable 
so we would have to move it above and remove the condition